examples/tutorials/README.md

```diff
@@ -27,28 +30,48 @@ practitioners. By enabling the growth of open source tools for drug discovery,
-* [Part 9: Creating a high fidelity model from experimental data](09_Creating_a_high_fidelity_model_from_experimental_data.ipynb)
-* [Part 10: Exploring Quantum Chemistry with GDB1k](10_Exploring_Quantum_Chemistry_with_GDB1k.ipynb)
-* [Part 11: Learning Unsupervised Embeddings for Molecules](11_Learning_Unsupervised_Embeddings_for_Molecules.ipynb)
-* [Part 12: Predicting Ki of Ligands to a Protein](12_Predicting_Ki_of_Ligands_to_a_Protein.ipynb)
-* [Part 13: Modeling Protein Ligand Interactions](13_Modeling_Protein_Ligand_Interactions.ipynb)
-* [Part 14: Modeling Protein Ligand Interactions With Atomic Convolutions](14_Modeling_Protein_Ligand_Interactions_With_Atomic_Convolutions.ipynb)
-* [Part 15: Synthetic Feasibility Scoring](15_Synthetic_Feasibility_Scoring.ipynb)
-* [Part 16: Conditional Generative Adversarial Networks](16_Conditional_Generative_Adversarial_Networks.ipynb)
-* [Part 17: Training a Generative Adversarial Network on MNIST](17_Training_a_Generative_Adversarial_Network_on_MNIST.ipynb)
-* [Part 18: Using Reinforcement Learning to Play Pong](18_Using_Reinforcement_Learning_to_Play_Pong.ipynb)
-* [Part 19: Large Scale Chemical Screens](19_Large_Scale_Chemical_Screens.ipynb)
-* [Part 20: ConvertingDeepChem Models to TensorFlow Estimators](20_Converting_DeepChem_Models_to_TensorFlow_Estimators.ipynb)
-* [Part 21: Introduction to Bioinformatics](21_Introduction_to_Bioinformatics.ipynb)
-* [Part 22: Using HuggingFace + Transfer Learning for Toxicity Predictions](22_Transfer_Learning_With_HuggingFace_tox21.ipynb)
+The tutorial is organized as follows:
 
+### Introduction to DeepChem
+The following tutorials covers the core features of DeepChem. Covering these
+tutorials will help you in getting started with DeepChem for machine learning. The later
+tutorials discuss about using DeepChem for specific applications.
 
+* [1 The Basic Tools of the Deep Life Sciences](The_Basic_Tools_of_the_Deep_Life_Sciences.ipynb)
+* [2 Working With Datasets](Working_With_Datasets.ipynb)
+* [3 An Introduction to MoleculeNet](An_Introduction_To_MoleculeNet.ipynb)
+* [4 Creating Models with Tensorflow and Pytorch](Creating_Models_with_TensorFlow_and_PyTorch.ipynb)
+* [5 Introduction to Graph Convolutions](Introduction_to_Graph_Convolutions.ipynb)
+* [6 Working With Splitters](Working_With_Splitters.ipynb)
+* [7 Advanced Model Training](Advanced_Model_Training.ipynb)
+* [8 Putting Multitask Learning to Work](Putting_Multitask_Learning_to_Work.ipynb)
+* [9 Conditional Generative Adversarial Networks](Conditional_Generative_Adversarial_Networks.ipynb)
+* [10 Training a Generative Adversarial Network on MNIST](Training_a_Generative_Adversarial_Network_on_MNIST.ipynb)
+* [11 Introduction to Model Interpretability](Introduction_to_Model_Interpretability.ipynb)
+* [12 Uncertainty in Deep Learning](Uncertainty_In_Deep_Learning.ipynb)
+* [13 Creating a high fidelity model from experimental data](Creating_a_high_fidelity_model_from_experimental_data.ipynb)
```

Comment:
Should we move this one to the Molecular ML section?

Reply:
I think that creating a high fidelity model is common to all sections. But I am not very sure of it. If it is can be generalized to other sections, we can keep it here otherwise move to molecular ML section.

Reply:
Ok sounds good! Let's leave it here for now